Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/stitch_borders.h

```diff
@@ -172,8 +172,8 @@ collect_duplicated_stitchable_boundary_edges
 
   typedef CGAL::dynamic_face_property_t<int>                        Face_property_tag;
   typedef typename boost::property_map<PM, Face_property_tag>::type Face_cc_map;
-  Face_cc_map cc;
-  std::size_t num_component;
+  Face_cc_map cc = 0;
+  std::size_t num_component = 0;
```

Comment:
Does this compile?! `Face_cc_map` is a property map and the initialization `Face_cc_map cc = 0;` does not look right.